usgs-details.html

```diff
@@ -34,6 +34,59 @@
         var BASE_URL = 'https://waterservices.usgs.gov/nwis/iv/';
         var REQUEST_URL = BASE_URL + '?format=json' + '&sites=' + SITE_ID;
 
+        var METRICS = {
+            temperature: {
+                code: '00010',
+                units: '°F',
+                transform: function(c) {
+                    var f = c * 9 / 5 + 32;
```

Comment:
The statement follows standard [JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence) and [Mathematical](https://en.wikipedia.org/wiki/Order_of_operations) precedence, and the formula itself is [quite standard](http://www.rapidtables.com/convert/temperature/how-celsius-to-fahrenheit.htm). Using parentheses where they are not required may give the intuition that something other than expected is happening, which is why I left this as is.

But perhaps readability can be improved by removing the spaces around `/`:

```js
var f = c * 9/5 + 32;
```